### PR TITLE
Add analysis for signature strings

### PR DESCRIPTION
**Pull request by snoe**
Addresses #1123

### CODE REVIEW

File: test/clj_kondo/analysis_test.clj [hunk 1]
```diff
@@ -326,6 +326,34 @@
         :to clojure.core}]
      var-usages)))
 
+(deftest ^:test-refresh/focus analysis-signatures-test
```
Comment (borkdude) [inline test/clj_kondo/analysis_test.clj:329]: :test-refresh/focus seems to be a tooling-specific thing, can you remove this?

Comment (snoe) [reply test/clj_kondo/analysis_test.clj]: this, and the other comments in the issue should be resolved now.

**Comment by borkdude** [comment]
@snoe I think the docs needs to be updated to `:arglists`.

**Comment by ericdallo** [comment]
Hey @borkdude thanks for the merge!
We are using [this branch](https://github.com/clojure-lsp/clojure-lsp/pull/261) with clj-kondo snapshot, would you mind generating another snapshot (or the same version) with the change from this PR? :)

**Comment by borkdude** [comment]
@ericdallo This happens automatically on CI:

https://app.circleci.com/pipelines/github/clj-kondo/clj-kondo/3270/workflows/9cc42c28-2935-4136-8553-698c7dc7ee46/jobs/12634

It's probably better to use the "dated" versions than `SNAPSHOT`:

`2020.12.13-20210117.170045-33`

You can find these numbers in the build.

**Comment by ericdallo** [comment]
Perfect, thank you!

**Pull request merged**